earthspy/earthspy.py

```diff
@@ -1065,7 +1075,7 @@ def merge_rasters(self) -> None:
                 # write mosaic
                 with rasterio.open(date_output_filename, "w", **output_meta) as dst:
                     dst.write(mosaic)
-
+                    dst.update_tags(**{k:self.metadata[date][0][k] for k in ["id"]})
```

Comment:
could you add the **kwargs in a variable before the context? Let's name it `id` or so

Reply:
Do I put it in the with or just before it ?

Reply:
Right before it is best I think!